packages/components/molecules/f-card-with-content/src/components/CardWithContent.vue

```diff
@@ -1,32 +1,107 @@
 <template>
-    <div
-        :class="$style['c-cardWithContent']"
-        data-test-id="cardWithContent">
-        I am a CardWithContent Component (GB)
-    </div>
+    <card-component
+        has-outline
+        is-page-content-wrapper
+        data-test-id="cardWithContent"
+        :class="$style['c-cardWithContent']">
+        <div
+            v-if="$slots.icon"
+            :class="$style['c-cardWithContent-icon']">
+            <slot name="icon" />
+        </div>
+
+        <h1
+            v-if="cardHeading"
+            :class="$style['c-cardWithContent-heading']">
+            {{ cardHeading }}
+        </h1>
+
+        <p
+            v-if="cardDescription"
+            :class="$style['c-cardWithContent-description']">
+            {{ cardDescription }}
+        </p>
+
+        <f-button
```

Comment:
Could these need to link anywhere potentially?

If so, should there be a way of defining an `href` for these links/buttons? (or were you thinking it would get handled slightly differently through the emitters?)

Reply:
Yes, I did originally have the `href`s, but looking at how it works in f-checkout they [already have an event handler](https://github.com/justeat/fozzie-components/blob/master/packages/components/pages/f-checkout/src/components/Error.vue#L30) so thought it would be easier to stick to that pattern, and it's more flexible too.

Reply:
I would check how accessible this is potentially - seems a bit strange to me to use events to redirect if it’s going to a url (if it’s not a form submission or something like that). 

Also, does that mean the buttons would do nothing if there was any sort of issue with the JS?

Reply:
I agree and my preference would have been to use links. Checkout's error page button currently does some extra stuff as well, like removing a menu-related cookie before handling the redirect.

I understand that links would be the preference for semantic HTML, but "Go back button" should still be easily interpreted by screen reader users?

Reply:
You can use a link and still perform some operations using a `click` handler I believe?

Reply:
Have updated to use an object which contains the text, handles `href` and `to`, and I've left in the event emitters